crml/cennzx-spot/src/impls.rs

```diff
@@ -92,7 +92,7 @@ pub(crate) mod impl_tests {
 	type CoreAssetCurrency = mock::CoreAssetCurrency<Test>;
 	type TradeAssetCurrencyA = mock::TradeAssetCurrencyA<Test>;
 	type FeeAssetCurrency = mock::FeeAssetCurrency<Test>;
-	type TestFeeExchange = FeeExchange<CennzXSpot>;
+	type TestFeeExchange = FeeExchange<u32, u128>;
```

Comment:
The purpose of introducing a type which could get CennzXSpot as the generic type was that we only specify the type of asset id and balance in one place for CennzxSpot and FeeExchange uses the same types. It would prevent from specifying different types in different places incidentally. Aligned to the DRY principle.

Reply:
Not essential for this PR